testing/functional/tests/cypress/integration/institute/03-institution-profile.spec.ts

```diff
@@ -0,0 +1,141 @@
+import LoginInstituteObject from "../../page-objects/institute-objects/LoginInstituteObject";
+import DashboardInstituteObject from "../../page-objects/institute-objects/DashboardInstituteObject";
+import InstitutionProfileObject from "../../page-objects/institute-objects/InstitutionProfileObject";
+
+describe("Institute Profile", () => {
+  const loginInstituteObject = new LoginInstituteObject();
+  const dashboardInstituteObject = new DashboardInstituteObject();
+  const institutionObject = new InstitutionProfileObject();
+
+  const url = Cypress.env("instituteURL");
+  const username = Cypress.env("bceid");
+  const password = Cypress.env("password");
+
+  beforeEach(() => {
+    cy.visit(url);
+    cy.intercept("GET", "**/bceid-account").as("bceidAccount");
```

Comment:
I believe that the process of having a user authenticated will be reused many times therefore it can be shared in a common method.